akamai/cache-client_test.go

```diff
@@ -17,6 +20,8 @@ import (
 	"github.com/letsencrypt/boulder/test"
 )
 
+var log = blog.UseMock()
```

Comment:
Any reason to define this as a top level var? It ends up being redefined elsewhere in this file a handful of times.

Reply:
(Also random thought: Using a single mock log doesn't mix well with running tests in parallel in general. I don't think we do any tests for certain log lines in these unit tests but if we were this could lead to those being rather flakey)